src/app/functions/getComponentAsString.tsx

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ReactElement } from "react";
+
+export type Props = {
+  component: ReactElement;
+};
+
+export const getComponentAsString = async ({ component }: Props) => {
```

Comment:
What's the reason we need this wrapper, rather than directly calling `renderToString`? Could you add that as a comment?

Reply:
Added the following comment in [e3f0628](https://github.com/mozilla/blurts-server/pull/3137/commits/e3f0628d0cbc5b90255404d9fa5709bb0b694cfd):

> If a component imports `react-dom/server` directly Next.js throws a compilation error `ReactServerComponentsError` for perf and security reasons.